src/styles/mixins.scss

```diff
@@ -13,3 +13,14 @@
     font-size: $font-size-md-pct;
   }
 }
+
+@mixin popoverButton {
+  &,
+  &:hover,
+  &:active:focus,
```

Comment:
Her skulle det kanskje bare stå `active`? For du nevner allerede `focus` under...

Reply:
Men når jeg setter det slik som det står så er alt transparent når jeg trykker på den også. Så tror ikke jeg kan endre dette? Prøv med knappen øverst i filter

Reply:
Du har rett... jeg må visst lære meg css bedre :-P